src/Components/SmartComponents/Reports/CustomReportFilter.js

```diff
@@ -162,6 +163,7 @@ const CustomReportFilter = ({ filterName, filterData, setFilterData }) => {
     const filterCvssWrapper = (
         <Select
             variant="panel"
+            ouiaId="filterCvss"
```

Comment:
same as above + do it also for `filterCheckboxes` and create unique ouia ids for every `filterContent` case